.env.sample

```diff
@@ -10,5 +10,9 @@ SITE_URL=
 # The value for this variable is using the same as SITE_URL unless you use an external CDN
 PUBLIC_URL=
 
+# The BROWSER where the development server will be open
```

Comment:
Change to: The browser that will be use to open the application when running the development server